io.catenax.request_for_quotation/1.0.0/RequestForQuotation.ttl

```diff
@@ -0,0 +1,584 @@
+
+:billOfMaterial a samm:Property ;
+   samm:preferredName "bill of material"@en ;
+   samm:description "RFQ-Configuration: reference to the single level bill of material as built"@en ;
+   samm:characteristic :BillOfMaterialCharacteristic .
+
+:manufacturingDomains a samm:Property ;
+   samm:preferredName "manufacturing domains"@en ;
+   samm:description "RFQ-Configuration: domain of required process to manufacture the part (might be different per part)"@en ;
+   samm:characteristic :SetOfManufacturingDomainCharacteristic .
+
+:manufacturingMethods a samm:Property ;
+   samm:preferredName "manufacturing methods"@en ;
+   samm:description "RFQ-Configuration: method how the part should be manufactured, related to manufacturing domain"@en ;
+   samm:characteristic :SetOfManufacturingMethodCharacteristic .
+
+:postProcesses a samm:Property ;
+   samm:preferredName "post processes"@en ;
+   samm:description "RFQ-Configuration: after manufacturing typically post processes are necessary, depending on the manufacturing method"@en ;
+   samm:characteristic :SetOfPostProcessCharacteristic .
+
+:billOfProcess a samm:Property ;
+   samm:preferredName "bill of process"@en ;
+   samm:description "RFQ-Configuration: list of processes to manufacture the part(s)"@en ;
+   samm:characteristic :BillOfProcessCharacteristic .
+
+:generalTolerance a samm:Property ;
+   samm:preferredName "general tolerance"@en ;
+   samm:description "RFQ-Configuration: selection of defined tolerance classes"@en ;
+   samm:characteristic :GeneralToleranceCharacteristic ;
+   samm:exampleValue "ISO 2768-1 (m), Rough DIN ISO 2768 cL, Medium DIN ISO 2768 mK, Fine DIN ISO 2768 fH" .
+
+:additionalQualityRequirements a samm:Property ;
+   samm:preferredName "additional quality requirements"@en ;
+   samm:description "RFQ-Configuration: specific requirements to guarantee quality"@en ;
+   samm:characteristic :AdditionalQualityRequirementsCharacteristic ;
+   samm:exampleValue "premium quality check, standard quality check" .
+
+:testPartRequired a samm:Property ;
+   samm:preferredName "test part required"@en ;
+   samm:description "RFQ-Configuration: makes only sens for batches"@en ;
+   samm:characteristic :TestPartRequiredCharacteristic ;
+   samm:exampleValue false .
+
+:quantity a samm:Property ;
+   samm:preferredName "Quantity"@en ;
+   samm:description "Quantity of which the child item is assembled into the parent item. In general it is '1' for serialized parts."@en ;
+   samm:characteristic :QuantityCharacteristic .
+
+:lastDeliveryDate a samm:Property ;
+   samm:preferredName "last delivery date"@en ;
+   samm:description "RFQ-Configuration: Date of required and last acceptable delivery"@en ;
+   samm:characteristic :DateCharacteristic .
+
+:firstDeliveryDate a samm:Property ;
+   samm:preferredName "first delivery date"@en ;
+   samm:description "RFQ-Configuration: Date of desired, first, but not must delivery"@en ;
+   samm:characteristic :DateCharacteristic .
```

Comment:
Please also provide the example value here e.g. '   samm:exampleValue "2023-05-22"^^xsd:date .'
same for `lastDeliveryDate`